packages/strapi-icons/assets/icons/Grid.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think @gu-stav updated icons to be 24px now 👀

Reply:
Should we be defining a width and height here? Wouldn't it be more versatile for the parent to declare that? Or is that gonna break a lot of things?

Reply:
I think this is fine: all icons do have it at the moment and as long as there is a viewBox attribute I think we are safe.

@ronronscelestes I didn't update the size - only removed padding on some.